great_expectations/dataset/base.py

```diff
@@ -2700,16 +2699,16 @@ def expect_column_kl_divergence_to_be_less_than(self,
         Returns:
             A JSON-serializable expectation result object.
 
-            Exact fields vary depending on the values passed to :ref:`output_format <output_format>` and
+            Exact fields vary depending on the values passed to :ref:`result_format <result_format>` and
             :ref:`include_config`, :ref:`catch_exceptions`, and :ref:`meta`.
 
         Notes:
             These fields in the result object are customized for this expectation:
             ::
 
                 {
-                  "true_value": (float) The true KL divergence (relative entropy)
-                  "summary_obj": {
+                  "observed_value": (float) The true KL divergence (relative entropy)
+                  "details": {
```

Comment:
This is a massive number of changes. You've been very thorough, and I'm impressed.